### PR TITLE
Add a simple dashboard example.

### PR DESCRIPTION
**Pull request by vicky5124**
This PR adds an example for a bot dashboard, using RillRate

	modified:   .github/workflows/ci.yml
	modified:   Cargo.toml
	modified:   Makefile.toml
	modified:   examples/README.md
	new file:   examples/e15_simple_dashboard/Cargo.toml
	new file:   examples/e15_simple_dashboard/Makefile.toml
	new file:   examples/e15_simple_dashboard/example video.mp4
	new file:   examples/e15_simple_dashboard/src/main.rs

### CODE REVIEW

File: examples/e15_simple_dashboard/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+
+    true
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
+    env::set_var(
+        "RUST_LOG",
+        // TODO: If you are going to copy this to your crate, update the crate name in the string
+        // with the name of the create you are using it with.
```
Comment (arqunis) [inline examples/e15_simple_dashboard/src/main.rs]:
```suggestion
        // with the name of the crate you are using it with.
```

**Review by arqunis — CHANGES_REQUESTED** [review]

**Comment by nickelc** [comment]
I think the 5.6 MB big example video shouldn't be included and hosted elsewhere.

**Comment by arqunis** [comment]
I agree. I should have touched upon that when I was submitting my review. Git isn't built for binary files; its job is to manage history of text files, particularly source code. If the video is committed into the repository, it will be forever there, even if we remove it later, unless we alter history.

**Comment by vicky5124** [comment]
The example video has been deleted on 1b9f819

**Review by arqunis — APPROVED** [review]

**Pull request merged**